README.md

```diff
@@ -1,241 +1,32 @@
 # Account Abstraction SDK (aa-sdk)
 
-⚠️ This repo is actively being developed and certain features might not be fully implemented yet or are subject to change ⚠️
+The `aa-sdk` is a type-safe and performant TypeScript library built on top of [viem](https://viem.sh/) to provide ergonomic methods for sending user operations, sponsoring gas, and deploying smart contract accounts. It handles all the complexity of ERC-4337 under the hood to make account abstraction simple.
```

Comment:
can you add a clear CTA and/or callout section that inks to the docs landing page?

Reply:
Fixed it